src/fileio/debugout.win.cpp

```diff
@@ -15,8 +15,8 @@ class basic_debugbuf : public std::basic_stringbuf<CharT, TraitsT> {
 	}
 protected:
 	int sync() {
-		output_debug_string(str().c_str());
-		str(std::basic_string<CharT>()); // Clear the string buffer
+		output_debug_string(this->str().c_str());
+		this->str(std::basic_string<CharT>()); // Clear the string buffer
```

Comment:
Really weird bug preventing me from compilation. Test this in xCode and let me know if it breaks anything.

Reply:
That's impossible, this code isn't even used in Xcode. So there's nothing to worry about.

Reply:
Awesome!